.circleci/config.yml

```diff
@@ -14,7 +14,7 @@ jobs:
           command: PYTHONHASHSEED=42 /opt/python/cp36-cp36m/bin/python setup.py test --parallel=4
   image-tests-mpl153:
     docker:
-      - image: astropy/image-tests-py35-mpl153:1.0
+      - image: astropy/image-tests-py35-mpl153:1.1
```

Comment:
@mhvk - can you try changing to ``1.2``? I think that should hopefully fix it.

Reply:
OK, changed (merged into last commit).